Mathlib/Tactic/ToAdditive.lean

```diff
@@ -293,6 +296,7 @@ partial def transformDeclAux
     range := ← getDeclarationRange (← getRef)
     selectionRange := ← getDeclarationRange ref
   }
+  MetaM.run' <| Term.TermElabM.run' <| Term.addTermInfo' ref trgDecl.value!
```

Comment:
```suggestion
(Term.addTermInfo' ref (← mkConstWithLevelParams tgtDecl.name)).run'.run'
```
You don't want to use `ConstantInfo.value!` because that will show the actual definition of the term rather than the constant itself (which triggers special support in the server to show the doc string etc.)